config_src/drivers/nuopc_cap/mom_ocean_model_nuopc.F90

```diff
@@ -132,7 +132,7 @@ module MOM_ocean_model_nuopc
 !> The ocean_state_type contains all information about the state of the ocean,
 !! with a format that is private so it can be readily changed without disrupting
 !! other coupled components.
-type, public :: ocean_state_type ; private
+type, public :: ocean_state_type ; 
```

Comment:
This is also needed, because:
```
/Users/heinzell/work/ufs-bundle/ufs-bundle-update-from-develop-20231212/ufs-weather-model/MOM6-interface/MOM6/config_src/drivers/nuopc_cap/mom_cap_methods.F90:533:65:

  533 |           isc, iec, jsc, jec, ocean_grid%ke, ocean_state%MOM_CSp%T, ocean_grid, rc=rc)
      |                                                                 1
Error: Component 'mom_csp' at (1) is a PRIVATE component of 'ocean_state_type'
/Users/heinzell/work/ufs-bundle/ufs-bundle-update-from-develop-20231212/ufs-weather-model/MOM6-interface/MOM6/config_src/drivers/nuopc_cap/mom_cap_methods.F90:543:65:

  543 |           isc, iec, jsc, jec, ocean_grid%ke, ocean_state%MOM_CSp%S, ocean_grid, rc=rc)
      |                                                                 1
Error: Component 'mom_csp' at (1) is a PRIVATE component of 'ocean_state_type'
/Users/heinzell/work/ufs-bundle/ufs-bundle-update-from-develop-20231212/ufs-weather-model/MOM6-interface/MOM6/config_src/drivers/nuopc_cap/mom_cap_methods.F90:553:65:

  553 |           isc, iec, jsc, jec, ocean_grid%ke, ocean_state%MOM_CSp%u, ocean_grid, rc=rc)
      |                                                                 1
Error: Component 'mom_csp' at (1) is a PRIVATE component of 'ocean_state_type'
/Users/heinzell/work/ufs-bundle/ufs-bundle-update-from-develop-20231212/ufs-weather-model/MOM6-interface/MOM6/config_src/drivers/nuopc_cap/mom_cap_methods.F90:563:65:

  563 |           isc, iec, jsc, jec, ocean_grid%ke, ocean_state%MOM_CSp%v, ocean_grid, rc=rc)
      |                                                                 1
Error: Component 'mom_csp' at (1) is a PRIVATE component of 'ocean_state_type'
```